includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -2040,7 +2040,7 @@ static function() {
 				$new_styles = str_replace( '@media only screen', '@media only screen and (prefers-color-scheme: dark)', $styles );
 				// Allow for rules to override the light theme related rules.
 				$new_styles = str_replace( '.is-dark-theme.is-dark-theme', ':root', $new_styles );
-				$new_styles = str_replace( '.respect-color-scheme-preference.is-dark-theme body', '.respect-color-scheme-preference body', $new_styles );
+				$new_styles = str_replace( '.respect-color-scheme-preference.is-dark-theme body', '.respect-color-scheme-preference:not(#_) body', $new_styles );
```

Comment:
The use of the ID selector is going to make this _too_ specific:

<img width="813" alt="Screen Shot 2021-03-12 at 13 27 29" src="https://user-images.githubusercontent.com/134745/111000347-bbf80b80-8336-11eb-9260-0d2c6e71cd88.png">

To retain the same specificity, a class selector need to be used. Either this can be done via `:not(._)` or it could be done by duplicating the class name.

```suggestion
				$new_styles = str_replace( '.respect-color-scheme-preference.is-dark-theme body', '.respect-color-scheme-preference:not(._) body', $new_styles );
```

![image](https://user-images.githubusercontent.com/134745/111000468-f661a880-8336-11eb-99fe-4baada98591d.png)